boot/zephyr/boards/conexio_stratus.conf

```diff
@@ -11,9 +11,6 @@ CONFIG_BOOT_MAX_IMG_SECTORS=256
 
 # MCUboot serial recovery
 CONFIG_MCUBOOT_SERIAL=y
-CONFIG_BOOT_SERIAL_DETECT_PORT="GPIO_0"
```

Comment:
drop this board patch
this is a unknown board.

Reply:
Dropped. Btw. should we keep configuration of non-zephyr board in here?